R/espn_league.R

```diff
@@ -0,0 +1,246 @@
+    "DL" = 11,
+    "CB" = 12,
+    "S" = 13,
+    "DB" = 14,
+    "DP" = 15,
+    "HC" = 19
+  )
+}
+
+#' @noRd
+#' @seealso \url{https://github.com/cwendt94/espn-api/blob/master/espn_api/football/constant.py}
+.espn_stat_map <- function() {
+  c(
+    "3" = "passingYards",
+    "4" = "passingTouchdowns",
+
+    "19" = "passing2PtConversions",
+    "20" = "passingInterceptions",
+
+    "24" = "rushingYards",
+    "25" = "rushingTouchdowns",
+    "26" = "rushing2PtConversions",
+
+    "42" = "receivingYards",
+    "43" = "receivingTouchdowns",
+    "44" = "receiving2PtConversions",
+    "53" = "receivingReceptions",
+
+    "72" = "lostFumbles",
+
+    "74" = "madeFieldGoalsFrom50Plus",
+    "77" = "madeFieldGoalsFrom40To49",
+    "80" = "madeFieldGoalsFromUnder40",
+    "85" = "missedFieldGoals",
+    "86" = "madeExtraPoints",
+    "88" = "missedExtraPoints",
+
+    "89" = "defensive0PointsAllowed",
+    "90" = "defensive1To6PointsAllowed",
+    "91" = "defensive7To13PointsAllowed",
+    "92" = "defensive14To17PointsAllowed",
+
+    "93" = "defensiveBlockedKickForTouchdowns",
+    "95" = "defensiveInterceptions",
+    "96" = "defensiveFumbles",
+    "97" = "defensiveBlockedKicks",
+    "98" = "defensiveSafeties",
+    "99" = "defensiveSacks",
+
+    "101" = "kickoffReturnTouchdown",
+    "102" = "puntReturnTouchdown",
+    "103" = "fumbleReturnTouchdown",
+    "104" = "interceptionReturnTouchdown",
+
+    "123" = "defensive28To34PointsAllowed",
+    "124" = "defensive35To45PointsAllowed",
+
+    "129" = "defensive100To199YardsAllowed",
+    "130" = "defensive200To299YardsAllowed",
+    "132" = "defensive350To399YardsAllowed",
+    "133" = "defensive400To449YardsAllowed",
+    "134" = "defensive450To499YardsAllowed",
+    "135" = "defensive500To549YardsAllowed",
+    "136" = "defensiveOver550YardsAllowed",
+
+    # Punter Stats
+    "140" = "puntsInsideThe10", # PT10
+    "141" = "puntsInsideThe20", # PT20
+    "148" = "puntAverage44.0+", # PTA44
+    "149" = "puntAverage42.0-43.9", #PTA42
+    "150" = "puntAverage40.0-41.9", #PTA40
+
+    # Head Coach stats
+    "161" = "25+pointsWinMargin", #WM25
+    "162" = "20-24pointWinMargin", #WM20
+    "163" = "15-19pointWinMargin", #WM15
+    "164" = "10-14pointWinMargin", #WM10
+    "165" = "5-9pointWinMargin", # WM5
+    "166" = "1-4pointWinMargin", # WM1
+
+    "155" = "TeamWin", # TW
+
+    "171" = "20-24pointLossMargin", # LM20
+    "172" = "25+pointLossMargin" # LM25
+  )
+}
+
+#' @noRd
+.espn_is_qbtype <- function(league_endpoint) {
+  position_map <- .espn_position_map()
+  qb_pos <- position_map["QB"]
+  op_pos <- position_map["OP"]
+  qb_count <- league_endpoint$content$settings$rosterSettings$lineupSlotCounts[[qb_pos]]
+  op_count <- league_endpoint$content$settings$rosterSettings$lineupSlotCounts[[op_pos]]
+
+  qb_type <- dplyr::case_when(
+    qb_count == 1 & op_count < 1 ~ "1QB",
+    qb_count == 1 & op_count == 1 ~ "2QB/SF",
+    TRUE ~ "2+QB/SF"
+  )
+
+  list(
+    count = qb_count,
```

Comment:
I think qb_count should be reported as a string (i.e. https://github.com/dynastyprocess/ffscrapr/blob/main/R/sleeper_league.R#L70) 

this will report 1-2 for your league, 2 for leagues that specifically have 2 QBs and not a super flex etc

Reply:
no worries on the test having env-vars in it